src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -534,6 +545,10 @@ class VAMap extends Component {
     const coords = this.props.currentQuery.position;
     const position = [coords.latitude, coords.longitude];
     const facilityLocatorMarkers = this.renderFacilityMarkers();
+    const externalLink =
+      currentQuery.facilityType === LocationType.CC_PROVIDER
+        ? urgentCareLink
+        : otherToolsLink;
```

Comment:
Seems like this could be defined outside of the methods similar to either link.

Reply:
The individual links are static and so can be defined anywhere, but which link gets displayed needs to be determined during the render based on what's currently selected in the dropdown.